motr/io_nw_xfer.c

```diff
@@ -1729,18 +1789,18 @@ static int nw_xfer_tioreq_map(struct nw_xfer_request           *xfer,
 	int                         rc;
 	struct m0_fid               tfid;
 	const struct m0_fid        *gfid;
-	struct m0_op_io     *ioo;
+	struct m0_op_io            *ioo;
 	struct m0_rpc_session      *session;
 	struct m0_pdclust_layout   *play;
 	struct m0_pdclust_instance *play_instance;
-	enum m0_pool_nd_state       device_state;
-	enum m0_pool_nd_state       device_state_prev;
+	enum m0_pool_nd_state       dev_state;
```

Comment:
What is the reason for renaming it to dev_state, as lot of references for device_state are still there in this file like in documentation and in other functions ?

Reply:
It seems shorter and more idiomatic.